command/seal_migration_test.go

```diff
@@ -231,7 +257,7 @@ func TestSealMigration(t *testing.T) {
 
 		var resp *api.SealStatusResponse
 		unsealOpts := &api.UnsealOpts{}
-		for _, key := range keys {
+		for i, key := range keys {
```

Comment:
Compilation is currently failing for me with the message:
```
./seal_migration_test.go:260:7: i declared and not used
```

Reply:
Ah, I took out some debugging statements and didn't run the test after (since it was passing) -- those were the users of `i`. Will remove.